website/docs/wallet/deterministic.md

```diff
@@ -87,46 +87,21 @@ Where you'll see the following output
 INFO accounts: (wallet path) /Users/johndoe/Library/Eth2Validators/prysm-wallet-v2
 
 Showing **1** validator account
-View the eth1 deposit transaction data for your accounts by running `validator accounts list --show-deposit-data
+View the eth1 deposit transaction data for your accounts by running `validator accounts list
```

Comment:
There are some places where \` are replaced with `<code>` but others not, should they all be replaced or only some?

Reply:
it's because of how docusaurus works, ` is markdown and <code> will tell the system to enable mdx inside which will allow for things like the prysm version custom component. it shouldn't be a big deal to use one or the other unless you want to do something specific inbetween the tags

Reply:
> by running `validator accounts list

Last "\`" is missing after `list` ?